### PR TITLE
merge expansionhunter calls

### PR DESCRIPTION
**Pull request by jemten**
## PR checklist

This PR addresses some outstanding issues with the expansionhunter workflow.
- Adds chromosome lengths to the vcf 
- Rename sample in VCF. The original sample name in the vcf is the name of the bam file rather than the sample ID
- Splits multiallelic 
- Merges sample vcfs into a case vcf 
- compress and index

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR> -stub`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR> -stub`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/call_repeat_expansions.nf [hunk 1]
```diff
@@ -2,26 +2,78 @@
...
+        )
+        RENAMESAMPLE_EXP ( BCFTOOLS_REHEADER_EXP.out.vcf )
+        TABIX_EXP_RENAME ( RENAMESAMPLE_EXP.out.vcf )
+
+        // Split multi allelelic
+        SPLIT_MULTIALLELICS_EXP (
+            RENAMESAMPLE_EXP.out.vcf.join(TABIX_EXP_RENAME.out.tbi),
+            fasta
+        )
+        //TABIX_EXP_SPLIT ( SPLIT_MULTIALLELICS_EXP.out.vcf )
```
Comment (ramprasadn) [inline subworkflows/local/call_repeat_expansions.nf:46]: I guess this comment/line can be removed? 😅

Comment (jemten) [reply subworkflows/local/call_repeat_expansions.nf]: oops missed that one

**Comment by jemten** [comment]
anymore comments @ramprasadn?

**Review by ramprasadn — APPROVED** [review]
Nope! Looks good 👍🏻 🚀

**Pull request merged**